src/Squidex/Areas/Api/Controllers/UI/Models/UISettingsDto.cs

```diff
@@ -3,26 +3,12 @@
 // ==========================================================================
 //  Copyright (c) Squidex UG (haftungsbeschränkt)
 //  All rights reserved. Licensed under the MIT license.
-// ==========================================================================
-
-using System.ComponentModel.DataAnnotations;
+// ==========================================================================using System.ComponentModel.DataAnnotations;
```

Comment:
Should this line be removed since it is commented out?

Reply:
Fixed